app/bundles/CampaignBundle/Tests/Executioner/Helper/EventRedirectionHelperTest.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CampaignBundle\Tests\Executioner\Helper;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Mautic\CampaignBundle\Entity\Campaign;
+use Mautic\CampaignBundle\Entity\Event;
+use Mautic\CampaignBundle\Executioner\Helper\EventRedirectionHelper;
+use Mautic\CoreBundle\Test\IsolatedTestTrait;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class EventRedirectionHelperTest extends TestCase
```

Comment:
I know it seems silly but all classes should be final by default unless there is a reason for them not to.
```suggestion
final class EventRedirectionHelperTest extends TestCase
```

Reply:
Fixed.